SPECS/apparmor/apparmor.spec

```diff
@@ -354,9 +355,10 @@ make DESTDIR=%{buildroot} install
 %exclude %{perl_archlib}/perllocal.pod
 
 %changelog
-* Sat May 09 00:20:37 PST 2020 Nick Samson <nisamson@microsoft.com> - 2.13-10
-- Added %%license line automatically
-
+*   Mon Sep 28 2020 Daniel McIlvaney <damcilva@microsoft.com> - 2.13-11
```

Comment:
There shouldn't be a `-` before the version.

Reply:
```suggestion
*   Mon Sep 28 2020 Daniel McIlvaney <damcilva@microsoft.com> 2.13-11
```